src/MoBi.Presentation/Presenter/EditParametersInContainerPresenter.cs

```diff
@@ -162,8 +161,10 @@ private ParameterDTO dtoFor(IParameter parameter)
 
       private void showParameters()
       {
-         var parametersToShowDTO = _allParametersDTO.Where(shouldShowParameter);
+         var parametersToShowDTO = _allParametersDTO.Where(shouldShowParameter).ToList();
          _view.BindTo(parametersToShowDTO);
+         setupEditPresenter(parametersToShowDTO.FirstOrDefault()?.Parameter);
```

Comment:
I always select the first parameter. In theory we could try to see if the selected parameter is still in the list so we do not reselect but the ref is not saved so I don't want to open a big can of worms